gobblin-runtime/src/main/java/gobblin/scheduler/JobScheduler.java

```diff
@@ -169,7 +169,7 @@ protected void startUp()
     LOG.info("Starting the job scheduler");
 
     try {
-      this.scheduler.awaitRunning(1, TimeUnit.SECONDS);
+      this.scheduler.awaitRunning(30, TimeUnit.SECONDS);
```

Comment:
curious: did you notice anything that takes time to bootstrap or you increased the awaitRunning preemptively because we have added a few components?